tests/common/test_event.py

```diff
@@ -239,7 +270,7 @@ def test_save_event_message_with_non_ascii_characters(self):
             event_str = MonitorHandler.collect_event(os.path.join(self.tmp_dir, tld_file))
             event_json = json.loads(event_str)
 
-            self.assertEqual(len(event_json["parameters"]), 8)
+            self.assertEqual(len(event_json["parameters"]), 9)
```

Comment:
why did the count increment?

Reply:
This tests counts the existing telemetry parameters before `sys_info` is added. Since I moved the `ContainerId` parameter from the `sys_info` parameter group to be populated at event creation time, it reflects as an additional parameter in the first stage.